hack/create_module_image.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
-# This script has the following argument: ci, for example:
+# This script has the following argument:
```

Comment:
Is this script in use by release pipeline too?

Reply:
In prow job `pull-btp-manager-module-build`